src/Domain/IndividualNeeds/INGateway.php

```diff
@@ -86,11 +86,38 @@ public function queryINBySchoolYear(QueryCriteria $criteria, $gibbonSchoolYearID
         return $this->runQuery($query, $criteria);
     }
 
+    public function queryIndividualNeedsPersonDescriptors(QueryCriteria $criteria)
+    {
+      $query = $this
+        ->newQuery()
+        ->from('gibbonINPersonDescriptor')
+        ->innerJoin('gibbonAlertLevel','gibbonAlertLevel.gibbonAlertLevelID = gibbonINPersonDescriptor.gibbonAlertLevelID')
+        ->cols([
+          'gibbonINPersonDescriptor.gibbonINPersonDescriptorID',
+          'gibbonINPersonDescriptor.gibbonPersonID',
+          'gibbonINPersonDescriptor.gibbonINDescriptorID',
+          'gibbonINPersonDescriptor.gibbonAlertLevelID',
+          'gibbonAlertLevel.gibbonAlertLevelID'
+        ]);
+
+      $criteria->addFilterRules([
+        'gibbonPersonID' => function($query,$gibbonPersonID)
+        {
+          return $query
+            ->where('gibbonINPersonDescriptor.gibbonPersonID = :gibbonPersonID')
+            ->bindValue('gibbonPersonID',$gibbonPersonID);
+        }
+      ]);
+
+      return $this->runQuery($query,$criteria);
+    }
+
     public function queryIndividualNeedsDescriptors(QueryCriteria $criteria)
     {
         $query = $this
             ->newQuery()
             ->from('gibbonINDescriptor')
+            ->orderBy(['gibbonINDescriptor.sequenceNumber'])
```

Comment:
Anytime we have a $criteria object, it's best to not explicitly set the orderBy in the query, as this overrides the ability to use the front-end column sorting by clicking the column names. The alternative is to call sortBy() on the $criteria object as it's created.